script.js

```diff
@@ -1,4 +1,98 @@
+var generateRandomInteger = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+
+  return diceNumber;
+};
+var totalGamePlayed = 0;
+var totalWin = 0;
+var totalLose = 0;
+var totalDraw = 0;
+var generateRandomPlay = function () {
+  number = generateRandomInteger();
+  if (number == 1) play = "stone";
+  if (number == 2) play = "paper";
+  if (number == 3) play = "scissor";
+  console.log(play);
+  return play;
+};
+var paperScissorStone = function (input) {
+  if (input !== "stone" && input !== "scissor" && input !== "paper") {
+    return `please input "scissor","paper", or "stone"`;
+  } else {
+    totalGamePlayed += 1;
+    console.log(`total game played`);
+    console.log(totalGamePlayed);
+
+    if (input == "stone") return stonePlay();
+    else if (input == "paper") return paperPlay();
+    else if (input == "scissor") return scissorPlay();
+  }
+};
+var stonePlay = function (input) {
```

Comment:
generally for functions naming, it should start with a verb if your function does something. In this case, since your function checks for winning conditions, perhaps the function could be named something more descriptive such as checkWinningForStonePlayed